maintainers/maintainer-list.nix

```diff
@@ -1541,6 +1541,12 @@
     githubId = 576355;
     name = "Bas van Dijk";
   };
+  BattleCh1cken = {
```

Comment:
Put this specific change in a separate commit. 
No need to open a pull-request.

Reply:
The other changes can be squashed in a single commit.

Reply:
Alright. Thanks for helping me!

Reply:
Alrighty. All done.